docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -4,7 +4,671 @@ sidebar_position: 2
 ---
 
 import ComponentIntroductionWarthog from "/components/introduction_warthog.mdx";
+import Support from "/components/support.mdx";
+
+## Introduction
```

Comment:
Is there a way to prevent this heading from showing up on the summary page? Eg. looking at https://docs.clearpathrobotics.com/docs/category/husky, it shows "Introduction" as part of the "User Manual" bubble.

Reply:
Likely, but I plan to remove the summary pages anyway. Refer to the Redux documentation which also uses Docusaurus:
https://redux-toolkit.js.org/introduction/getting-started

Their configuration does not allow the User to navigate to the summary pages.

Reply:
Sounds good. Makes sense.